resources/views/docs/1/digging-deeper/child-process.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Child Processes
+order: 700
+---
+
+# Child Processes
+
+Child Processes allow your application to spin up managed processes, forked from your app's main process. This is great
+for long-running processes that you want to interact with repeatedly during the life of your application.
+ 
```

Comment:
We start off with a significant chunk of text. Maybe this is a good spot for a simple code example? To give the reader some idea what it's about.

Reply:
added in #54

Reply:
added in [52c8ebd](https://github.com/NativePHP/nativephp.com/pull/54/commits/52c8ebdfbf0639f5e39b02c2f6a0c24a8ffb463b)